### PR TITLE
change semantics

### PR DESCRIPTION
**Pull request by wdwatkins**

### CODE REVIEW

**Comment by wdwatkins** [comment]
@ldecicco-USGS is there an example for facebook?  we could put that in too.

**Comment by wdwatkins** [comment]
just gonna change in the viz for now

File: inst/templates/semantics.mustache [hunk 1]
```diff
@@ -9,11 +9,11 @@
   <meta property="og:locale" content="en_US" />
 
   <meta name="twitter:card" content="summary_large_image" />
-  <meta name="twitter:site" content="{{ twitter }}" />
+  <meta name="twitter:site" content="{{ url }}" />
```
Comment (jiwalker-usgs) [inline inst/templates/semantics.mustache]: `twitter:site` is supposed to be a twitter username `@user`

**Comment by mwernimont** [comment]
looks simple enough

File: inst/templates/semantics.mustache (hunk 1 above)
Comment (wdwatkins) [reply inst/templates/semantics.mustache]: wasn't sure on that

Comment (wdwatkins) [reply inst/templates/semantics.mustache]: Is there a template somewhere for what facebook should look like?

**Pull request merged**